lib/ansible/module_utils/facts.py

```diff
@@ -780,10 +781,6 @@ def get_distribution_OpenWrt(self, name, data, path):
         if release:
             self.facts['distribution_release'] = release.groups()[0]
 
-    def get_distribution_Archlinux(self, name, data, path):
-        self.facts['distribution'] = 'Archlinux'
-        self.facts['distribution_version'] = data
-
     def get_distribution_Alpine(self, name, data, path):
         self.facts['distribution'] = 'Alpine'
         self.facts['distribution_version'] = data
```

Comment:
As discussed in IRC: I guess Alpine has content in it's release-file that is relevant to have in distribution_version. Arch does not (as far as I know).